templates/restaurant/list.html.twig

```diff
@@ -32,4 +122,26 @@
 {% block scripts %}
   {{ encore_entry_script_tags('search-address') }}
   {{ encore_entry_script_tags('restaurant-list') }}
+
+  <script>
+    $('input[type=radio]').on('click', function (e) {
```

Comment:
This selector is too wide, at least the filters should be wrapped in another top-level element to narrow the selector.